installing/installing-preparing.adoc

```diff
@@ -19,7 +19,9 @@ If you want to install and manage {product-title} yourself, you can install it o
 
 * Alibaba Cloud
 * Amazon Web Services (AWS) on x86_64 instances
+ifndef::openshift-origin[]
 * Amazon Web Services (AWS) on arm64 instances
+endif::[]
```

Comment:
`endif::[]` -> `endif::openshift-origin[]` 
I know that it still works either way, but we prefer having them labeled